ui/src/lib/helpers.tsx

```diff
@@ -337,3 +337,9 @@ export const getFullDOIsStrings = (text: string) =>
 export const getDOIsFromText = (text: string) => text.match(/(10\.[0-9a-zA-Z]+\/(?:(?!["&\'])\S)+)\b/g) || [];
 
 export const validateDOI = (value: string) => /(10\.[0-9a-zA-Z]+\/(?:(?!["&\'])\S)+)\b/.test(value);
+
+export const validateEmail = (email: string): Boolean => {
+    const regex = /[a-zA-Z0-9'_\\-\\.\\+]+@[a-zA-Z0-9'_\\-\\.]+[.][a-zA-Z0-9'_\\-\\.]+/;
+    if (email.match(regex)) return true;
```

Comment:
I know email validation it's a vast topic, but this validation accepts inputs like "test test test@test.com" (see the screenshot below) and then, the backend returns an error (which we're not displaying) and the user has no clue what happened. 

I think we also need to display any error messages coming back from the backend in the same way we are displaying the ones for frontend validation.

The regex needs to capture the entire input value, that means it needs to capture the start "^" and the end "$". Currently, this validation just tells if the string contains a potential email address. You can use "regex.test(email)" instead of "email.match(regex)" in this case because "match" returns an array with all matches it finds, and you don't need that array, you just need to know if it's valid or not.

![image](https://user-images.githubusercontent.com/48569671/211004583-eed26bed-c34c-404d-a249-f3b9e44ad4b6.png)